data/reusables/user-settings/password-authentication-deprecation.md

```diff
@@ -1 +1 @@
-When Git prompts you for your password, enter your personal access token (PAT) instead.{% ifversion not ghae %} Password-based authentication for Git has been removed, and using a PAT is more secure.{% endif %} For more information, see "[Creating a personal access token](/github/authenticating-to-github/creating-a-personal-access-token)."
+Password-based authentication for Git will no longer work, as it has been removed in favor of more secure authentication methods. Instead use a credential helper like [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/f5bd57908b17e7772dde33d84c3a2634fb9d4768/README.md#git-credential-manager) or a [personal access token](/github/authenticating-to-github/creating-a-personal-access-token)."
```

Comment:
What about for GHAE? The original text had "if not GHAE" conditionals?

Also drop the "," from ".. no longer work, as it has .."

Reply:
Well-spotted, thank you.